<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ControlGroup">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Vertical Speed</string>
    </property>
    <widget class="QPushButton" name="RadialOut">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>180</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Radial Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ResetIntegral">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>110</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Integral</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ClimbRate">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="ZeroOut">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Output">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="digitCount">
      <number>4</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="Enable">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="decrement">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="increment">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
